### DISCUSSION TITLE
Trying to Uninstall Udisks from Steam Deck

### DISCUSSION THREAD

**Discussion by FrontoOptimo** [storaged-project/udisks#1276] [General]
I been trying to get rid of this on my steam deck for months. Is their any codes to use to uninstall it  from linux steam deck

**Comment by FrontoOptimo** [comment]
![image](https://github.com/storaged-project/udisks/assets/137258852/f6e72a02-a2bf-4888-b9a8-39e336926dab)

**Comment by tbzatek** [comment]
We're upstream, please follow up with your distribution. Also, the screenshot looks correct.

**Reply by FrontoOptimo** [reply]
Im confused
